components/centraldashboard/public/components/main-page.css

```diff
@@ -168,6 +150,10 @@ paper-tab[link] a:hover {
     text-decoration: none;
 }
 
+app-header {
+    box-shadow: 0 2px 6px 1px rgba(232, 234, 237, 0.5), 0 1px 3px 1px rgba(60, 64, 67, 0.15);
```

Comment:
Could we try to match this to:
```css
var(--paper-elevation-<some number>)
```

Reply:
None of the paper-style [shadow-elevations](https://github.com/PolymerElements/paper-styles/blob/master/shadow.js) match @ajayalfred 's specification. 

Here is a screenshot of applying `--shadow-elevation-2dp, the least aggressive elevation available in paper-styles.
![Screenshot from 2019-05-06 09-41-40](https://user-images.githubusercontent.com/6835846/57229033-2b866c80-6fe3-11e9-9891-05a8ddd92aa5.png)